scripts/modal.js

```diff
@@ -0,0 +1,34 @@
+module.exports = function installModal () {
+  // Get the modal
+  const modal = document.getElementById('myModal')
```

Comment:
How would you feel about using a different ID here? Something like `keyboard-reference-modal` or something?